modulemd/include/private/modulemd-util.h

```diff
@@ -122,6 +334,22 @@ modulemd_validate_nevra (const gchar *nevra);
   while (0)
 
 
+/**
+ * MODULEMD_SETTER_GETTER_STRING_EXT:
```

Comment:
Note that this is the internal implementation for `MODULEMD_SETTER_GETTER_STRING` and `MODULEMD_SETTER_GETTER_STRING_STATIC` and that those convenience macros should be preferred.